docs/content/setup/manual-setup.md

```diff
@@ -1,41 +1,139 @@
+   See [below](#systemd-unit-example) for an example using systemd.
+
+## Upgrading
+
+!!! warning
+    Before you upgrade, **always read the release notes**.  
+    You can find them on our [releases page](https://hedgedoc.org/releases/).
 
 If you are upgrading HedgeDoc from an older version, follow these steps:
 
-1. Check if you meet the [requirements at the top of this document](#requirements-on-your-server).
-2. Verify which version you were running before and take a look at [migrations and breaking changes](../guides/migrations-and-breaking-changes.md) to see if additional steps, or configuration changes are necessary!
-3. Fully stop your old HedgeDoc server.
-4. `git pull` or unzip a new release in the directory.
+1. Check if you still meet the [requirements at the top of this document](#requirements-on-your-server).
+2. Ensure you read the [release notes](https://hedgedoc.org/releases/) of all versions between your current version
+   and the latest release.
+2. Fully stop your old HedgeDoc server.
+3. [Download](https://hedgedoc.org/latest-release/) the new release and extract it over the old directory.  
+   <small>If you use Git, you can checkout the new tag with e.g. `git checkout 1.7.2`</small>
 5. Run `bin/setup`. This will take care of installing dependencies. It is safe to run on an existing installation.
-6. Build front-end bundle by `yarn run build` (use `yarn run dev` if you are in development).
-7. It is recommended to start your server manually once: `NODE_ENV=production yarn start`, this way it's easier to see warnings or errors that might occur (leave out `NODE_ENV=production` for development).
+6. *:octicons-light-bulb-16: If you used the release tarball for 1.7.0 or newer, this step can be skipped.*  
+   Build the frontend bundle by running `yarn run build`.  
+   <small>For debugging purposes you can instead build an unoptimized bundle with source-maps using `yarn run dev`</small>
+7. It is recommended to start your server manually once:
+   ```shell
+   NODE_ENV=production yarn start
+   ```
+   This way it's easier to see warnings or errors that might occur.  
+   <small>You can leave out `NODE_ENV=production` for development.</small>
 8. You can now restart the HedgeDoc server!
+
+## Systemd Unit Example
+Using the unit file below, you can run HedgeDoc as a systemd service.
+
+!!! warning
+    - In this example, you must configure HedgeDoc using the `config.json` file and the 
+    `production` key.
+    - Make sure the user and group `hedgedoc` exists and has appropriate permissions in the
+    directory you installed HedgeDoc in or change the `User` and `Group` settings in the unit
+    file.
+    - Make sure `WorkingDirectory` points to the directory you installed HedgeDoc in.
+    - Make sure `ReadWritePaths` contains all directories HedgeDoc might write to. This may
+    include the `public/uploads` folder if you configure local storage. If you use SQLite, you
```

Comment:
```suggestion
    include the `public/uploads` folder if you configured local storage. If you use SQLite, you
```

Reply:
I believe "configure" also works here

Reply:
I think it should carry the tense in the list point above, where you used 'installed'. But this is not the hill I want to die on.